samples/drivers/firmware/arm_scmi/README.rst

```diff
@@ -0,0 +1,77 @@
+.. _arm_scmi_shell:
+
+ARM SCMI shell sample
+#####################
+
+Overview
+********
+
+This sample demonstrates the usage of ARM SCMI. This sample
```

Comment:
nit: tautology, you can just replace "This sample" with "It" in second sentence.

Reply:
fixed